lib/datasource/pod/index.ts

```diff
@@ -32,13 +40,26 @@ function shardParts(lookupName: string): string[] {
 
 function releasesGithubUrl(
   lookupName: string,
-  opts: { account: string; repo: string; useShard: boolean }
+  opts: {
+    hostURL: string;
+    account: string;
+    repo: string;
+    useShard: boolean;
+    useSpecs: boolean;
+  }
```

Comment:
This is a bit of a nitpick, but what do you think about extracting this complex type?

Reply:
I am ok extracting the options from the complex type and passing them as parameters. I was just following the current method which already had `opts: { account: string; repo: string; useShard: boolean }`. Maybe be what I should do is pull out hostURL and add useSpecs to the options. so it will look like this:

```TypeScript
function releasesGithubUrl(
  lookupName: string,
  opts: {
    account: string;
    repo: string;
    useShard: boolean;
    useSpecs: boolean;
  },
  hostURL?: string,
):
```

Reply:
Okay, let's leave it for now. It was a nitpick anyway 😄